source/includes/fact-verifier-buildIndexes.rst

```diff
@@ -0,0 +1,9 @@
+
+If you start sync with verification enabled and ``buildIndexes``
+set to ``never``, the ``/start`` endpoint returns an error if
+``mongosync`` finds a TTL collection on the source cluster.
+
+To sync TTL collections without building indexes on the
+destination cluster, you must start sync with the :ref`verifier
+disabled <c2c-verifier-disabled>`.
```

Comment:
Based on the current staging link, this ref isn't rendering properly. It shows up as `:ref`verifier disabled <c2c-verifier-disabled>`.`

Reply:
Fixed